x-pack/legacy/plugins/uptime/common/constants/index_names.ts

```diff
@@ -6,5 +6,4 @@
 
 export const INDEX_NAMES = {
   HEARTBEAT: 'heartbeat-8*',
-  HEARTBEAT_STATES: 'heartbeat-states-8*',
```

Comment:
we are not using this anymore?

Reply:
I don't think we've been using it for several releases.